lib/resources/r4/communication.yaml

```diff
@@ -0,0 +1,193 @@
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.priority
+  action: create
+  note: Millennium only supports Routine and High Prioirities.  Any Priority given other than Routine will be translated to High.  If not provided, the Communication will be treated as Routine.
+  example: |
+    {
+      "priority": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/communiucation.priority",
+            "code": "routine"
+          }
+        ]
+      }
+    }
+- name: subject
+  required: 'No'
+  cardinality: 0..1
+  type: Reference(Patient)
+  description: The Patient that is the Subject to the Communicatiion
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.subject
+  action: update
+  note: Millennium only supports Patient References for the Subject
+  example: |
+    {
+      "subject": {
+        "reference": "Patient/9836548",
+        "type": "Patient"
+      }
+    }
+- name: topic
+  required: 'yes'
+  cardinality: 0..1
+  type: code
+  description: The topic of the Communication.  Simalar to the subject of an email
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.topic
+  note: Millennium Message Center only supports Free Text Topics.
+  example: |
+    {
+      "topic" : {
+         "text": "Patient Consult Request"
+      }
+    }
+  action:
+    - create
+
+
+- name: encounter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference(Encounter)
+  description: The specific Patient Encounter associated to the Communcation
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.encounter
+  example: |
+    {
+      "encounter" : {
+        "reference" : "Encounter/89363836",
+        "type" : "Encounter"
+      }
+    }
+  action:
+    - create
+
+- name: recipient
+  required: 'Yes'
+  cardinality: 1..*
+  type: Reference(Practitioner/Group)
+  description: The Practitioner or Pool (Group) that the communication is intended to reach
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.recipient
+  note: At least one recipient needs to be provided
+  example: |
+    {
+      "recipient": [
+        {
+          "reference" : "Practitioner/6373634",
+          "type" : "Practitioner"
+        },
+        {
+          "reference" : "Group/44466678",
+          "type" : "Group"
+        }
+      ]
+    }
+  action:
+      - create
+
+- name: sender
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference
+  description: The Practitioner sending the Message
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.sender
+  note: Sender must be a Practitioner
+  example: |
+    {
+      "sender": {
+        "reference": "Practitioner/631923",
+        "type" : "Practitioner"
+      }
+    }
+  action:
+    - create
+
+- name: payload
+  required: 'Yes'
+  cardinality: 1..1
+  type: BackboneElement
+  description: The text of the communication
```

Comment:
Let's change this to The message content of the communication

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/b51e76543bb04c72ff83076cbc51694bdcb436f1#diff-f08ae1e4c063b9a36a22adaa43fecf21R169